include/mujoco/mjmacro.h

```diff
@@ -60,4 +60,21 @@
   }
 #endif
 
+//-------------------------- nullability check
+//----------------------------------------------------
+
+#ifdef __clang__
+#define NONNULL_ARG _Nonnull
```

Comment:
1. Please indent the `#ifdef` bodies.
2. I don't think we need NONNULL_ARG. Clang [supports](https://clang.llvm.org/docs/AttributeReference.html#id12) the GCC syntax.
3. Due to 2, let's rename `NONNULL_FUNC` -> `NONNULL` and `NULLABLE_ARG` -> `NULLABLE`.
4. I think this whole block belongs in mujoco.h and requires `#undef`s at the end

Reply:
Will do it and also I will restore the other unintentional changes that happened.